output/bremen/agency.txt

```diff
@@ -5,27 +5,21 @@ agency_id,agency_name,agency_url,agency_timezone,agency_lang,agency_phone,agency
 67234159,BremerhavenBus,http://hiposfer.com,Europe/Berlin,,,,
 94911646,Masemann,http://hiposfer.com,Europe/Berlin,,,,
 21921969,Weser-Ems_Bus,http://hiposfer.com,Europe/Berlin,,,,
-42445103,DB Fernverkehr AG,http://hiposfer.com,Europe/Berlin,,,,
```

Comment:
@mehdisadeghi why did so many agencies got deleted? I was expecting the same number of additions as deletion in this file (i.e. reordering)

Reply:
Hmm. Perhaps they where unused?